MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,961 @@
+<td>Specifies the type of the text search mode in ComboBox and AutoComplete.</td>
+</tr>
+
+<tr>
+<td>{{'[ValidationMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.ValidationMode.html)'| markdownify }}</td>
+<td>{{'[DataFormValidationMode]'| markdownify }}</td>
+<td>Defines the constants for ValidationMode that specifies when the user input should be validated for data object.</td>
+</tr>
+</table>
+
+## Events
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[AutoGeneratingDataFormItem](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html)'| markdownify }}</td>
+<td>{{'[GenerateDataFormItem]'| markdownify }}</td>
+<td>Occurs during the generation of data form item in the data form when AutoGenerateItems is true.</td>
+</tr>
+
+<tr>
+<td>{{'[Validating](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_Validating)'| markdownify }}</td>
+<td>{{'[ValidateProperty]'| markdownify }}</td>
+<td>Occurs when SfDataForm validates the user input.</td>
+</tr>
+
+<tr>
+<td>{{'[ValidationCompleted](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_ValidationCompleted)'| markdownify }}</td>
+<td>{{'[ValidateForm]'| markdownify }}</td>
+<td>Occurs when DataFormValidationMode is Manual to get the invalid DataFormItem’s details.</td>
+</tr>
+</table>
+
+## Methods
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[RegisterEditor](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_RegisterEditor_System_String_Syncfusion_XForms_DataForm_Editors_DataFormEditorBase_)'| markdownify }}</td>
+<td>{{'[RegisterEditor]'| markdownify }}</td>
+<td>Mthod to register DataFormEditor.</td>
+</tr>
+
+<tr>
+<td>{{'[Validate](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_Validate().html)'| markdownify }}</td>
+<td>{{'[Validate]'| markdownify }}</td>
+<td>Validates all the fields of DataFormViewItem</td>
+</tr>
+
+<tr>
+<td>{{'[Commit](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_Commit().html)'| markdownify }}</td>
+<td>{{'[Commit]'| markdownify }}</td>
+<td>Method to commit all properties underlying data object from the editor view.</td>
+</tr>
+
+<tr>
+<td>{{'[UpdateEditor](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_UpdateEditor_System_String_)'| markdownify }}</td>
+<td>{{'[UpdateEditor]'| markdownify }}</td>
+<td>Updates the value of the specific property in corresponding editor.</td>
+</tr>
+</table>
+
+## Upcoming Features
+
```

Comment:
## Upcoming Features

Floating label layout support
Keyboard interaction support for MacCatalyst, Android and iOS
View customization for group header.
Auto height support for DataFormViewItem
Programmatic scroll to a particular editor.
Themeing support for dataform.

Make better sentence for above

Reply:
Added upcoming features